src/main/java/org/candlepin/subscriptions/subscription/ActiveMQServiceConfiguration.java

```diff
@@ -51,6 +51,14 @@ public ActiveMQSslConnectionFactory activeMQSslConnectionFactory(
       ActiveMQProperties activeMQProperties) throws Exception {
     ActiveMQSslConnectionFactory factory = new ActiveMQSslConnectionFactory();
     factory.setExceptionListener(e -> log.error("Exception thrown in ActiveMQ connection", e));
+    if (StringUtils.hasText(activeMQProperties.getUser())) {
```

Comment:
I had to include these changes, so I can verify the changes using a remote ActiveMQ connection (from a container).